Failed to execute the command ‘make lib‘

Failed to execute the command ‘make lib‘
Failed to execute the command ‘make‘
Unable to compile source file


ERROR INFO:
make[3]: Leaving directory '/home/admin/K/src/lib'
test -f /home/admin/K/build-linux-x86_64/include/catch.h || (mkdir -p /home/admin/K/build-linux-x86_64/include                      \
&& curl -L https://github.com/catchorg/Catch2/releases/download/v2.13.7/catch.hpp \
-o /home/admin/K/build-linux-x86_64/include/catch.h                                                          )
make[3]: Nothing to be done for 'uv'.
make[3]: Leaving directory '/home/admin/K/src/lib'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   315  100   315    0     0    703      0 --:--:-- --:--:-- --:--:--   704

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make[3]: *** [Makefile:49: zlib] Error 2
make[3]: Leaving directory '/home/admin/K/src/lib'
make[2]: *** [Makefile:21: j0] Error 2
make[2]: Leaving directory '/home/admin/K/src/lib'
make[1]: *** [Makefile:18: lib] Error 2
make[1]: Leaving directory '/home/admin/K/src/lib'
make: *** [Makefile:131: lib] Error 2

--- a/src/lib/Makefile
+++ b/src/lib/Makefile
@@ -2,7 +2,7 @@ ERR      = *** Unexpected MAKELEVEL = 0.
 HINT     = This Makefile can't be used directly, consider cd ../.. before try again
 $(if $(subst 0,,${MAKELEVEL}),,$(warning $(ERR));$(error $(HINT)))
 
-V_ZLIB  = 1.2.11
+V_ZLIB  = 1.2.12
 V_SSL   = 3.0.0
 V_CURL  = 7.80.0
 V_SQL   = 3370000

